examples/cpp/configuration/PublisherApp.hpp

```diff
@@ -29,7 +29,7 @@
 
 #include "Application.hpp"
 #include "CLIParser.hpp"
-#include "ConfigurationPubSubTypes.hpp"
+#include "Configuration.hpp"
```

Comment:
Please, revert this change

Reply:
I think it is necessary to keep this change. We use it below to define a new instance of the class ConfigurationPubSubType

Reply:
Yeah, reverting means keeping `ConfigurationPubSubTypes.hpp` i.e not changing it to `Configuration.hpp`